packages/web-components/fast-components/src/default-palette.ts

```diff
@@ -96,101 +96,101 @@ export const neutralPalette: string[] = [
     "#FFFFFF",
-    "#FEFBFC",
-    "#FEF7FA",
-    "#FDF4F7",
-    "#FDF0F5",
-    "#FCECF2",
-    "#FBE8EF",
-    "#FBE5ED",
-    "#FAE1EA",
-    "#FADDE7",
-    "#F9D9E5",
-    "#F8D6E2",
-    "#F8D2E0",
-    "#F7CEDD",
-    "#F7CADA",
-    "#F6C7D8",
-    "#F5C3D5",
-    "#F5BFD2",
-    "#F4BBD0",
-    "#F3B8CD",
-    "#F3B4CB",
-    "#F2B0C8",
-    "#F2ACC5",
-    "#F1A9C3",
-    "#F0A5C0",
-    "#F0A1BD",
-    "#EF9DBB",
-    "#EF9AB8",
-    "#EE96B6",
-    "#ED92B3",
-    "#ED8EB0",
-    "#EC8BAE",
-    "#EC87AB",
-    "#EB83A8",
-    "#EA7FA6",
-    "#EA7CA3",
-    "#E978A1",
-    "#E9749E",
-    "#E8709B",
-    "#E76D99",
-    "#E76996",
-    "#E66593",
-    "#E66191",
-    "#E55E8E",
-    "#E45A8C",
-    "#E45689",
-    "#E35286",
-    "#E24F84",
-    "#E24B81",
-    "#E1477E",
-    "#E1437C",
-    "#E04079",
-    "#DF3C77",
-    "#DF3874",
-    "#DE3471",
-    "#DE316F",
-    "#DD2D6C",
-    "#DC2969",
-    "#DC2567",
-    "#DB2264",
-    "#DB1E62",
-    "#DA1A5F",
-    "#D4195C",
-    "#CD1859",
-    "#C71857",
-    "#C01754",
-    "#BA1651",
-    "#B3154E",
-    "#AD154B",
-    "#A71449",
-    "#A01346",
-    "#9A1243",
-    "#931240",
-    "#8D113D",
-    "#86103B",
-    "#800F38",
-    "#7A0F35",
-    "#730E32",
-    "#6D0D2F",
-    "#660C2D",
-    "#600B2A",
-    "#590B27",
-    "#530A24",
-    "#4D0921",
-    "#46081F",
-    "#40081C",
-    "#3B071A",
-    "#350617",
-    "#300615",
-    "#2B0513",
-    "#260511",
-    "#21040E",
-    "#1C030C",
-    "#000000",
+    "#FBFDFE",
+    "#F6FAFE",
+    "#F2F8FD",
+    "#EEF6FC",
+    "#E9F4FB",
+    "#E5F1FB",
+    "#E1EFFA",
+    "#DCEDF9",
+    "#D8EAF8",
+    "#D4E8F8",
+    "#CFE6F7",
+    "#CBE4F6",
+    "#C7E1F6",
+    "#C2DFF5",
+    "#BEDDF4",
+    "#BADAF3",
+    "#B6D8F3",
+    "#B1D6F2",
+    "#ADD4F1",
+    "#A9D1F0",
+    "#A4CFF0",
+    "#A0CDEF",
+    "#9CCAEE",
+    "#97C8EE",
+    "#93C6ED",
+    "#8FC4EC",
+    "#8AC1EB",
+    "#86BFEB",
+    "#82BDEA",
+    "#7DBAE9",
+    "#79B8E8",
+    "#75B6E8",
+    "#70B3E7",
+    "#6CB1E6",
+    "#68AFE5",
+    "#63ADE5",
+    "#5FAAE4",
+    "#5BA8E3",
+    "#56A6E3",
+    "#52A3E2",
+    "#4EA1E1",
+    "#499FE0",
+    "#459DE0",
+    "#419ADF",
+    "#3D98DE",
+    "#3896DD",
+    "#3493DD",
+    "#3091DC",
+    "#2B8FDB",
+    "#278DDB",
+    "#238ADA",
+    "#1E88D9",
+    "#1A86D8",
+    "#1683D8",
+    "#1181D7",
+    "#0D7FD6",
+    "#097DD5",
+    "#047AD5",
+    "#0078D4",
+    "#0075CF",
+    "#0072C9",
+    "#006FC4",
+    "#006CBE",
+    "#0069B9",
+    "#0066B4",
+    "#0063AE",
+    "#0060A9",
+    "#005CA3",
+    "#00599E",
+    "#005699",
+    "#005393",
+    "#00508E",
+    "#004D88",
+    "#004A83",
+    "#00477D",
+    "#004478",
+    "#004173",
+    "#003E6D",
+    "#003B68",
+    "#003862",
+    "#00355D",
+    "#003258",
+    "#002F52",
+    "#002B4D",
+    "#002847",
+    "#002542",
+    "#00223C",
+    "#001F36",
+    "#001B30",
+    "#00182B",
+    "#001525",
+    "#00121F",
+    "#000000"
```

Comment:
@freefaller69 this looks like a bad rebase. The default palette changed and this appears to be reverting it. See if you can work with @radium-v or @eljefe223 to revert this change.

Reply:
fixed